src/main/gpiiConnector.js

```diff
@@ -818,14 +769,36 @@ gpii.app.dev.gpiiConnector.qss.prepareMessageForQss = function (that, defaultQss
 gpii.app.dev.gpiiConnector.qss.distributeQssSettings = function (that, message) {
     var payload = message.payload || {},
         value = payload.value || {},
-        qssSettingControls = value.qssSettingControls || {};
+        preferences = value.preferences || {},
+        settingGroups = value.settingGroups || [],
+        updateDefaultValues = false;
 
-    if (gpii.app.gpiiConnector.isPrefSetUpdate(payload)) {
-        fluid.log("gpiiConnector.qss Controls to be sent: ", value.qssSettingControls);
+    if (gpii.app.gpiiConnector.isPrefSetUpdate(payload) && fluid.isValue(settingGroups[0])) {
+        var settingControls = settingGroups[0].settingControls;
+
+        // update settings values with the default values from the PSP Channel when Morphic start or reset
+        if (!fluid.isValue(preferences.contexts)) {
```

Comment:
I think this entire function is unnecessary, right? The values that came from the PSP channel are the ones that are already the defaults. I don't understand why you are making any update at all ....